### DISCUSSION TITLE
AI醫療應用案例：重症照護系統、智慧藥櫃

### DISCUSSION THREAD

**Discussion by lukiea** [1101-GS3073-C/Python-Community-GS3073-C#700] [General]
https://makerpro.cc/2021/03/the-improvement-of-artificial-intelligence-technology-and-its-application-in-medical-treatment/
 人們在醫療方面的需求愈發重要，有時候交給人類不一定準確，可以交由ai 來接手

**Comment by Valentinaa0214** [comment]
AI在醫療方面的幫助既能協助醫生診斷的準確性與效率，更能讓病人有更好的就診體驗。我很喜歡文章中提到的智慧藥櫃設計，每次看病拿藥都要等好久，希望這項技術能夠順利普及。

**Comment by yjpxi** [comment]
AI 已經被用來分析醫學影像，如X光片、CT掃描和MRI。AI能夠檢測到人眼難以察覺的細微變化，並且速度快、準確率高。
